apps/communications/dialer/test/marionette/lib/views/activities/views.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+/* global module */
+
+var ActivitiesAccessor = require('./accessors');
+
+//and list the rest of accessors needes, like the other POs to go I guess
+
+//var ComposerAccessor = require('../shared/composer_accessors');
+//var MenuAccessor = require('../shared/menu_accessors');
+//var Tools = require('../shared/tools');
+
+var appRoot = require('app-root-path');
+// TODO Change the path once requireFromApp becomes its own module
+var fromApp = require(appRoot +
+  '/shared/test/integration/require_from_app');
+
+function ActivitiesView(client) {
+  this.client = client;
+
+  this.accessors = new ActivitiesAccessor(client);
+  //this.composerAccessors = new ComposerAccessor(client);
+  //this.menuAccessors = new MenuAccessor(client);
+}
+
+ActivitiesView.prototype = {
+
+  //addContact: function(){
+  //  var addContactButton = this.accessors.addNewContact;
+  //  addContactButton.tap();
+
+  //  var Contacts = fromApp('contacts').require('lib/contacts');
+  //  var contacts = new Contacts(this.client);
+  //  contacts.switchToApp();
+  //  return contacts;
+  //}
+
+  tapCreateContactOption: function() {
+     //client.waitFor(function() {
+     //   return this.accessors.addNewContact.enabled();
+      //}.bind(this));
+      var button = this.accessors.addNewContact;
+      button.tap();
```

Comment:
Yes, it was not finished yet...I added:
var Contacts = fromApp('contacts').require('lib/contacts');
      var contacts = new Contacts(this.client);
      contacts.switchToCreateNewContactActivity();
      return contacts;
And it works...:)